apps/ensadmin/src/components/recent-registrations/types.ts

```diff
@@ -15,12 +15,59 @@ export interface Domain {
   };
 }
 
+/**
+ Data associated with a Registration event.
+ **/
+export interface LatestRegistration {
+  /**
+   * recently registered ENS name
```

Comment:
```suggestion
   * The registered ENS name
```

This data model should not be specific to a "recent registration". It should be general to any "registration"